app/controllers/charges_controller.rb

```diff
@@ -203,7 +210,8 @@ def create_square_payment_request(
       recipient: recipient,
       campaign: @campaign,
       metadata: metadata,
-      project: Project.find_by(id: project_id)
+      project: Project.find_by(id: project_id),
```

Comment:
nit: Project.find(...)

Reply:
What happens if there is no `project_id`?

Reply:
The caveat of find is that I think that is crashes if it doesn't exist, but find_by returns nil

Reply:
I didn't add this line but if a transaction has no project (e.g. single-seller GAM, donation), there wouldn't be a project so nil should be passed otherwise the transaction won't complete unless I'm mistaken

For future multi-seller GAMs that do not have a project, we are passing the campaign, which should be enough to create a charge?